openapi/provider_factory.go

```diff
@@ -174,6 +176,15 @@ func (p providerFactory) createTerraformProviderResourceMap() (map[string]*schem
 		return nil, err
 	}
 	for _, openAPIResource := range openAPIResources {
+		resourceName, err := p.getProviderResourceName(openAPIResource.getResourceName())
+		if err != nil {
+			return nil, err
+		}
+		if _, alreadyThere := resourceMap[resourceName]; alreadyThere {
```

Comment:
shall we check for ignored resources before or after duplicated ones?

Reply:
Probably before.  Good point.